Source/Core/CesiumTerrainProvider.js

```diff
@@ -532,17 +531,18 @@ define([
             extensionList.push('watermask');
         }
 
-        function tileLoader(tileUrl) {
-            return loadArrayBuffer(tileUrl, getRequestHeader(extensionList));
+        // TODO - remove later, this handles the deprecated throttleRequests parameter
```

Comment:
Remove the TODO and submit an issue to deprecate this.

Same comment throughout.